lib/Doctrine/MongoDB/EagerCursor.php

```diff
@@ -65,6 +65,7 @@ public function __construct(Cursor $cursor)
     public function count()
     {
         $this->initialize();
+
         return count($this->data);
```

Comment:
I dislike adding a feature (as minor as it might be) in an interface and then violate this interface in a child class. In this case, I'd rather remove the ```$foundOnly``` parameter in the cursor interface and only add it in those cursors that actually support it.